packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx

```diff
@@ -263,6 +264,12 @@ export const CommentsItem = ({ treeOptions, comment, nestingLevel=1, isChild, co
   if (!comment) {
     return null;
   }
+
+  const displayReviewVoting = 
+    reviewIsActive() &&
+    comment.reviewingForReview === REVIEW_YEAR+"" &&
```

Comment:
Stylistically, I don't like this string concatenation and then comparison, but eh.